byzcoin/README.md

```diff
@@ -14,6 +14,43 @@ them as seem fit.
 This document describes the part of ByzCoin that are implemented and how to
 use them. It should grow over time as more parts of the system are implemented.
 
+## Short summary of ByzCoin
+
+- ByzCoin is a **distributed ledger**: a database that is
+  distributed across several nodes, and where the authority to add changes
+  to the database is decentralized.
+- To ensure that updates to the database are strictly ordered, ByzCoin
+  uses a **blockchain** called a **Skipchain**: every node has its local copy of the ledger
+  (database). Every change to the ledger is packed into a block that is
```

Comment:
The novelties of skipchains are the following:
- delegation of trust from one block to the next: as the roster is stored in the block, the current nodes delegate trust to the next set of nodes
- finality using forwardlinks: where other blockchains use proof-of-work and longest chain to define the latest state of the system, skipchains never change and don't allow for split views (except twice in the 9cc3 skipchain, due to an error ;)
- short proofs: using only the hash of the genesis block (9cc3...), every node can return a proof that the latest block is valid by sending only the forward-links. In other systems you need to have your own trusted node follow the skipchain so you can be sure that the latest block is valid

and one thing that is not implemented, but describe in `Chainiac`:
- using multiple skipchains, where one holds the definition of the roster, while the other skipchain holds the actual transactions.

Reply:
that's great, I'm adding it to the `skipchain/README`